addons/crm/views/crm_lead_views.xml

```diff
@@ -71,21 +71,21 @@
                                     <a class="border-0 mx-1 p-0 mb-1 mb-md-0 btn btn-light"
                                         name="action_set_automated_probability"
                                         role="button" type="object"
-                                        invisible="is_automated_probability or won_status == 'lost'">
+                                        invisible="is_automated_probability or won_status == 'lost' or not id">
```

Comment:
instead you should hide/disable toot tip button here
https://github.com/odoo/odoo/blob/79668bb20e80d5f57812bee922a22fdc5e86bd80/addons/crm/static/src/views/crm_form/crm_pls_tooltip_button.xml#L4

```xml
t-attf-class="o_crm_pls_tooltip_button border-0 mx-1 p-0 mb-1 mb-md-0 btn btn-light {{!this.props.record.resId ? 'invisible': ''}}"

remove another changes to
```